examples/fashion_conv_dense.py

```diff
@@ -0,0 +1,99 @@
+import sys
+sys.path.append('../')
+from vulcanai2 import models, datasets, plotters
+from vulcanai2.models import ConvNet, DenseNet, SnapshotNet, BaseNetwork
+from vulcanai2.datasets import MultiDataset
+from vulcanai2.plotters.visualization import (compute_saliency_map, 
+                                              display_saliency_overlay,
+                                              display_receptive_fields,
+                                              display_confusion_matrix)
+
+import torch
+
+import torchvision.transforms as transforms
+from torch.utils.data import DataLoader, TensorDataset
+# from torchviz import make_dot
+
+
+sys.path.append('../')
+normalize = transforms.Normalize(mean=[x/255.0 for x in [125.3, 123.0, 113.9]],
```

Comment:
don't think this is actually used... something happened since the first iteration.

Reply:
Good catch! I just pushed a fix to clean up the script a bit more